<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRouteDataQuery</class>
 <widget class="QWidget" name="FormRouteDataQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>航线数据查询</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportButton">
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
